src/components/products-page/product-page-components/product-page-body.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ProductsData } from '../../../data/products-data';
+import { SingleProductCard } from './single-product-card';
+
+const ProductsPageBody = () => {
+	return (
+		<main className='product-main'>
+			{ProductsData.map((product) => {
+				return (
+					<SingleProductCard
+						key={product.id}
+						image={product.image}
+						image2={product.image2}
+						name={product.name}
+						price={product.price}
+						offer={product.offer}
+						inStock={product.inStock}
+					/>
+				);
```

Comment:
Rule of thumb: When more than 3 properties are there try restructuring. It'll look cleaner

Reply:
Will make this in the next update. Thanks!